api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -734,16 +783,49 @@ public void test_maybeSendAccessExpirationEmail_expired() throws MessagingExcept
     accessModuleService.updateCompletionTime(
         dbUser, DbAccessModuleName.RT_COMPLIANCE_TRAINING, expiredBy(oneHour));
 
-    userService.maybeSendAccessExpirationEmail(dbUser);
+    userService.maybeSendAccessTierExpirationEmails(dbUser);
+
+    // Registered Tier Expiration Email is sent
+    verify(mailService)
+        .alertUserAccessTierExpiration(dbUser, expirationTime, REGISTERED_TIER_SHORT_NAME);
+
+    // Controlled Tier Expiration Email is sent because RT Compliance is a requirement for CT Tier.
```

Comment:
It doesn't look like the user previously had CT access in this case, so an email should not be sent.